### PR TITLE
OSDOCS#10999: HCP introducing MCE and a relationship between HCP, MCE, and ACM

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.13+

Issue: https://issues.redhat.com/browse/OSDOCS-10999 also covers https://issues.redhat.com/browse/OSDOCS-10237

Link to docs preview: https://82536--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/index.html#hcp-mce-acm-dependency-intro_hcp-overview

QE review:
- [x] QE has approved this change. 
- [x] SME has approved this change.
- [x] PM has approved this change.

Additional information:

### CODE REVIEW

File: modules/hcp-mce-acm-relationship-intro.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/index.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-mce-acm-dependency-intro_{context}"]
+= Relationship between {hcp}, {mce-short}, and {rh-rhacm}
+
+You can configure {hcp} by using the {mce}. The multicluster engine is an integral part of {rh-rhacm-first} and is enabled by default. {rh-rhacm} cluster lifecycle defines the process of creating, importing, managing, and destroying Kubernetes clusters across various infrastructure cloud providers, private clouds, and on-premises data centers.
```
Comment (berenss) [inline modules/hcp-mce-acm-relationship-intro.adoc:9]: ...and is enabled by default **with {rh-rhacm}**. {mce-short} cluster lifecycle defines the process ...

**Review by berenss — APPROVED** [review]

File: modules/hcp-mce-acm-relationship-intro.adoc (hunk 1 above)
Comment (xenolinux) [reply modules/hcp-mce-acm-relationship-intro.adoc]: Thanks for the review! Addressed.

**Comment by rokej** [comment]
@xenolinux Should we add the relationship between MCE and HCP described by the second paragraph and the second diagram in https://github.com/rokej/hypershift-addon-operator/blob/introduction_doc/docs/intro.md?

**Comment by xenolinux** [comment]
> @xenolinux Should we add the relationship between MCE and HCP described by the second paragraph and the second diagram in https://github.com/rokej/hypershift-addon-operator/blob/introduction_doc/docs/intro.md?

Oh sorry I forgot to mention on slack. I did not include the second para because that information is already added in other module here -- https://docs.openshift.com/container-platform/4.17/hosted_control_planes/index.html. But if it's okay to include this info at both the places. I can get it in!!

**Review by rokej — APPROVED** [review]

File: modules/hcp-mce-acm-relationship-intro.adoc [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+====
+
+You can deploy {product-title} clusters by using two different control plane configurations: standalone or hosted control planes. The standalone configuration uses dedicated virtual machines or physical machines to host the control plane. With {hcp} for {product-title}, you create control planes as pods on a management cluster without the need for dedicated virtual or physical machines for each control plane.
+
+.{rh-rhacm} and the {mce-short} introduction diagram
+image::rhacm-flow.png[{rh-rhacm} and the {mce-short} introduction diagram]
+
+Review the following management cluster topologies:
+
+* An {product-title} cluster as a management cluster managed by {mce-short}.
```
[the remark this answers is not in the document]
Comment (rokej) [reply modules/hcp-mce-acm-relationship-intro.adoc:30]: I am not sure what lines 28 ~ 32 are for. Is this needed in this introduction topic? 

However @dtthuynh you are right that `ACM cluster as a management cluster managed by MCE` is not supported scenario.

[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/hcp-mce-acm-relationship-intro.adoc:30]: Ack. Removed those lines.

**Pull request merged**